src/shared/components/mutationTable/column/clonal/ClonalColumnFormatter.spec.tsx

```diff
@@ -102,19 +114,25 @@ describe('ClonalColumnFormatter', () => {
         testExpectedClonalElementProperties(
             sampleToClonalElement['S001'],
             'S001',
-            'yes',
+            'clonal',
```

Comment:
I'm a bit puzzled. Here the value is lowercase, but you have the `clonal` value defined few lines above all uppercase.

Reply:
@zhx828 Yes, so this is because we are getting data that's all uppercased from the backend ("CLONAL'", "SUBCLONAL", "INDTERMINATE") and using the enum we map it to the lowercase equivalent (or in the future we could map it to another display string). 

This display string gets passed to the ClonalElement to be used inside the tooltip. 

e.g In the database and backend api response we'll get mutation with ascn.clonal = `CLONAL`
which when displayed will have a tooltip displaying `clonal`